CMakeLists.txt

```diff
@@ -142,6 +147,12 @@ add_compile_options(
 
     # Fix Fatal Error C1128: number of sections exceeded object file format limit : compile with /bigobj on SerializeEmpire.cpp
     $<$<CXX_COMPILER_ID:MSVC>:/bigobj>
+
```

Comment:
Unused junk, delete it.

Reply:
This is my careless fault, I will revert the commit and fix it.